python/lsst/ap/association/ssoAssociation.py

```diff
@@ -125,12 +125,11 @@ def run(self, diaSourceCatalog, solarSystemObjects, exposure):
         maskedObjects = self._maskToCcdRegion(
             solarSystemObjects,
             exposure,
-            solarSystemObjects["Err(arcsec)"].max())
+            solarSystemObjects["Err(arcsec)"].max()).copy()
```

Comment:
I don't think you want this `.copy()` statement. It is making a copy of the return value of `self._maskToCcdRegion`, which does not seem useful.

Reply:
self._maskToCcdRegion returns a copy-by-reference mask of solarSystemObjects, which I can't set a column of (i.e. `associated` later). Copying the dataframe makes it alterable.